backend/fleet_management/tests/test_crypto.py

```diff
@@ -0,0 +1,62 @@
+from secrets import randbits
+
+from django.conf import settings
+from rest_framework.test import APISimpleTestCase
+from fleet_management.crypto import (
+    sign,
+    verify,
+    inverse_of,
+    is_prime,
+    find_prime,
+    find_p_q_phi,
+    find_pair_of_keys,
+)
+
+
+class CryptoTest(APISimpleTestCase):
+    def setUp(self) -> None:
+        self.n_tests = 1000
+        self.keys = [find_pair_of_keys() for _ in range(self.n_tests)]
```

Comment:
I know 1000 is a small number, but instaed of saving that to the list we could in `test_sign_and_verify` run:
```
for _ in range(self.n_tests):
    pub, priv = find_pair_of_kyes()
   ....
```